lib/daylight/associations.rb

```diff
@@ -21,8 +21,8 @@ def association_path(reflection)
         "#{prefix}#{collection_name}/#{member_id}/#{reflection.name}.#{extension}"
       end
 
-      def call_remote(remoted_method, model)
-        response = get(remoted_method)
+      def call_remote(remoted_method, model, verb, args)
```

Comment:
my only argument for verb is that method in this context loses meaning because there's also `remoted_method` as well as the next would be `self.method(method)` which would be kinda confusing